src/console-services.yml

```diff
@@ -191,50 +415,98 @@
+  description: 'WAF & Shield - Protects Against DDoS Attacks and Malicious Web Traffic'
+  url: 'https://console.aws.amazon.com/shield/home'
+
+- command: singlesignon
+  description: 'Single Sign-On - Manage Single Sign-On (SSO) Access to AWS Accounts and Business Applications'
+  url: 'https://console.aws.amazon.com/singlesignon/home'
+
+- command: snowball
+  description: 'Snowball - Large Scale Data Transport'
+  url: 'https://console.aws.amazon.com/snowball/home'
+
 - command: sns
-  description: 'SNS - Push Notification Service'
+  description: 'Simple Notification Service - SNS Pub/Sub, Mobile Push and SMS'
   url: 'https://console.aws.amazon.com/sns/home'
 
 - command: sqs
-  description: 'SQS - Message Queue Service'
+  description: 'Simple Queue Service - SQS Managed Message Queues'
   url: 'https://console.aws.amazon.com/sqs/home'
 
 - command: stepfunctions
-  description: 'Step Functions - Coordinate components of distributed applications'
+  description: 'Step Functions - Coordinate Distributed Applications'
   url: 'https://console.aws.amazon.com/states/home'
 
 - command: storagegateway
   description: 'Storage Gateway - Hybrid Storage Integration'
   url: 'https://console.aws.amazon.com/storagegateway/home'
 
-- command: swf
-  description: 'SWF - Workflow Service for Coordinating Application Components'
-  url: 'https://console.aws.amazon.com/swf/home'
+- command: sumerian
+  description: 'Sumerian - Build VR, AR, and 3D applications'
+  url: 'https://console.aws.amazon.com/sumerian/home'
 
-- command: support
-  description: 'Support - Support the success and operational health of your AWS solutions'
-  url: 'https://console.aws.amazon.com/support/home'
+- command: systemsmanager
+  description: 'Systems Manager - a Central Place to View and Manage AWS Resources'
+  url: 'https://console.aws.amazon.com/systemsmanager/home'
+
+- command: transcribe
+  description: 'Transcribe - Powerful Speech Recognition'
+  url: 'https://console.aws.amazon.com/transcribe/home'
 
-- command: trustedadvisor
-  description: 'Trusted Advisor - Optimize Performance and Security'
-  url: 'https://console.aws.amazon.com/trustedadvisor/home'
+- command: translate
+  description: 'Translate - Translate Text in Real-time'
+  url: 'https://console.aws.amazon.com/translate/home'
 
 - command: vpc
   description: 'VPC - Isolated Cloud Resources'
   url: 'https://console.aws.amazon.com/vpc/home'
 
 - command: waf
-  description: 'WAF - Filter Malicious Web Traffic'
+  description: 'WAF & Shield - Protects Against DDoS Attacks and Malicious Web Traffic'
   url: 'https://console.aws.amazon.com/waf/home'
 
+- command: wellarchitectedtool
+  description: 'Well-Architected Tool - learn best practices, measure, and improve your workloads'
+  url: 'https://console.aws.amazon.com/wellarchitectedtool/home'
+
+- command: workdocs
+  description: 'WorkDocs - Secure Enterprise Storage and Sharing Service'
+  url: 'https://console.aws.amazon.com/workdocs/home'
```

Comment:
another 404... is it possible that these links have been removed in the 2 months that have ellapsed since the PR was posted? 😓

Reply:
This is a weird one. Not sure what happened here. Have replaced this with WorkLink